featuretools/primitives/standard/binary_transform.py

```diff
@@ -881,7 +881,10 @@ def __init__(self, commutative=False):
         self.commutative = commutative
 
     def get_function(self):
-        return np.divide
+        def divide_numeric(val1, val2):
```

Comment:
When primitives directly return a function, it tends to be faster. Is there no way to keep using `np.divide`?

Reply:
![image](https://user-images.githubusercontent.com/4307001/175331701-2440d2b9-c4d9-4ae4-9121-c8cfbc1d93b1.png)

Not sure it makes a difference in this particular case - perhaps because we are using optimized approaches in both cases (numpy vs pandas)?